src/utils/filterWorkspaces.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import multimatch from 'multimatch';
+import path from 'path';
+
+import Workspace from '../Workspace';
+import * as logger from '../utils/logger';
+
+type FilteringOpts = {
```

Comment:
So, this isnt great, wondering if you know a better way?
Flow yells because a flag can either be a string or a bool and supposedly (???) it can't coerce a bool to a string, so it yells on lines 16-19.

Adding this type here though, means you need to create a separate opts object any time you want to filter, that's pretty crap too.

Could we add explicit types for the options we know will be strings perhaps?

Reply:
Technically options could end up inside commands as strings or booleans since there is nothing to validate them... We need a better way to do that.

In the meantime I've just been doing stuff like:

```js
let cwd = typeof opts.cwd === 'string' ? opts.cwd : process.cwd();
```